content/sql/concepts/primary-keys/primary-keys.md

```diff
@@ -82,8 +82,20 @@ For example, a database of car parts will have to uniquely identify a row of par
 
 Depending on local regulations, a car may require an engine part ID and a body ID to be associated with a license plate. One solution might be adding more row information about the car, such as `left_door_ID`, `gearbox_ID`, etc. But then a specific car would have to be identified by two different aspects: its body and its engine.
 
+If you have created a table that doesn't have a primary key, you can alter it to add a one.
+
+The statement below creates a `PRIMARY KEY` constraint on the `ID` column on the `people` table:
```

Comment:
```suggestion
The statement below creates a `PRIMARY KEY` constraint on the `id` column on the `people` table:
```

Reply:
Please see line 35 in this same entry. It composes of the same info so this is a bit redundant. Please move this example below line 40 (after the pseudo code snippet) as an example. Could you also explain in one sentence as to why a table would be created without a primary key in the first place? Or why it's important to have a primary key?

Reply:
Sorry, just to clarify did you want to move line 87 (The statement below creates a `PRIMARY KEY`...) below the pseudo code snippet starting on line 89?

And would the sentence below be ok?:
It's important for a table to have a `PRIMARY KEY` to help prevent duplicated data by giving every row in the table a unqiue identfier, this also allows the user to find data quicker.

Reply:
Thanks for your patience @Rclarkeweb ! I helped to make that change ^^